clang.Dockerfile

```diff
@@ -27,9 +27,11 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 # Make clang the default compiler
-ENV CC /usr/bin/clang-6.0
-ENV CXX /usr/bin/clang++-6.0
-RUN ln -s /usr/bin/clang-format-6.0 /usr/bin/clang-format \
+ENV CC clang
+ENV CXX clang++
+RUN ln -s /usr/bin/clang-9 /usr/bin/clang \
+ && ln -s /usr/bin/clang++-9 /usr/bin/clang++ \
+ && ln -s /usr/bin/clang-format-6.0 /usr/bin/clang-format \
```

Comment:
Shouldn't this and the line below be linking `clang-format-9`

Reply:
Nice catch